backend/data.go

```diff
@@ -126,9 +126,9 @@ func (r QueryDataResponse) MarshalJSON() ([]byte, error) {
 
 // UnmarshalJSON will read JSON into a QueryDataResponse
 func (r *QueryDataResponse) UnmarshalJSON(b []byte) error {
-	iter := jsoniter.ParseBytes(jsoniter.ConfigDefault, b)
+	iter, _ := sdkjsoniter.ParseBytes(sdkjsoniter.ConfigDefault, b)
```

Comment:
since this is now returning an error, we should return it if it's not `nil`

Reply:
There are lots of places that returns error now. Trying to handle all of them will be quite hard. I handled a couple of them but let others stay. I'll handle errors in more places.

Reply:
I remember why I skipped this particular one. `ParseBytes` are just creating a new `Iterator` instance and nothing else. It is not literally parsing anything at that point. So error will always be nil. But there will be no harm in checking it for sure.